src/app/lib/README.md

```diff
@@ -100,6 +100,26 @@ Therefore to meet all requirements we implement the following CSS grid where gut
 - `6 + 2 of 0px` - is based on having a 0px column either side of UX grid so we can use CSS grid-gaps as the UX margin while allowing a grid item to be able to go "full bleed".
 - `6 + 2 of 8px` - between 400px and 599px we have an 8px column either side of the UX grid so that the grid-gap (8px) and additional column (8px) combined meet the 16px UX margin.
 
+### Nested grid usage
+
+As of [#1680](https://github.com/bbc/simorgh/pull/1680) `GridItem`s now also have their own internal grid which maps to the external grid they occupy.
```

Comment:
```suggestion
As of [#1680](https://github.com/bbc/simorgh/pull/1680) there are a new set of grid items called `NestedGridItem`s with varying sizes which correspond to the size of the containing `GridItem` (in order to match breakpoints / the space occupied parent grid)
```

Reply:
Thanks this clarifies my point below 👍 

https://github.com/bbc/simorgh/pull/1680/files#r286877770